mobile/native/src/ln_dlc/mod.rs

```diff
@@ -192,6 +192,8 @@ pub fn run(data_dir: String, seed_dir: String, runtime: &Runtime) -> Result<()>
 
         NODE.set(node);
 
+        event::publish(&EventInternal::Init("Backend initialised".to_string()));
```

Comment:
nit: How about `10101 is ready!`

Reply:
changed to `10101 is ready.` - I really think we're overusing the bangs, normal logs don't need to scream 😄